wpilibc/src/main/native/include/frc/simulation/BatterySim.h

```diff
@@ -52,8 +55,10 @@ class BatterySim {
   static units::volt_t Calculate(
       units::volt_t nominalVoltage, units::ohm_t resistance,
       std::initializer_list<units::ampere_t> currents) {
-    return nominalVoltage -
-           std::accumulate(currents.begin(), currents.end(), 0_A) * resistance;
+    return std::max(
```

Comment:
Please put the 0_V at the beginning of the argument list to be consistent with Java.